### PR TITLE
Fix: set min sdk version of sentry-android-fragment to API 14

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
https://mvnrepository.com/artifact/androidx.fragment/fragment and https://androidx.tech/artifacts/fragment/fragment/ state min API 14

## :bulb: Motivation and Context
Fix https://github.com/getsentry/sentry-java/issues/1607

## :green_heart: How did you test it?
CI

## :pencil: Checklist

- [X] I reviewed the submitted code
- [ ] I added tests to verify the changes
- [ ] I updated the docs if needed
- [X] No breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Fix: set min sdk version of sentry-android-fragment to API 14
```
Comment (marandaneto) [inline CHANGELOG.md:5]:
```suggestion
* Fix: set min sdk version of sentry-android-fragment to API 14 (#1608)
```

File: buildSrc/src/main/java/Config.kt [hunk 2]
```diff
@@ -33,7 +33,6 @@ object Config {
 
         val minSdkVersion = 14
         val minSdkVersionOkHttp = 21
-        val minSdkVersionFragment = 21
```
Comment (marandaneto) [inline buildSrc/src/main/java/Config.kt:36]: @wzieba do you remember why you've set 21 instead of 14?

File: buildSrc/src/main/java/Config.kt [hunk 3]
```diff
@@ -86,7 +85,7 @@ object Config {
 
         val coroutinesCore = "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3"
 
-        val fragment = "androidx.fragment:fragment-ktx:1.3.4"
+        val fragment = "androidx.fragment:fragment-ktx:1.3.5"
```
Comment (marandaneto) [inline buildSrc/src/main/java/Config.kt:88]: theres a critical bugfix from 1.3.4

File: buildSrc/src/main/java/Config.kt (hunk 2 above)
Comment (wzieba) [reply buildSrc/src/main/java/Config.kt:36]: I don't remember and I think it was rather not a deliberative decision - I think I've just copied the value from `minSdkVersionOkHttp`. Sorry!

Comment (marandaneto) [reply buildSrc/src/main/java/Config.kt:36]: not a problem at all, we're in a beta state so its the right moment to get it fixed :)
thanks again for the contribution @wzieba

**Review by maciejwalkowiak — APPROVED** [review]
LGTM

**Pull request merged**